front/src/pods/trainer/components/session.styles.ts

```diff
@@ -118,3 +118,45 @@ export const undoIcon = css`
     display: initial;
   }
 `;
+
+export const downButton = css`
```

Comment:
This is twice?

Reply:
renamed to downloadButton and downloadIcon